libpod/runtime_img.go

```diff
@@ -168,9 +168,7 @@ func (r *Runtime) newImageBuildCompleteEvent(idOrName string) {
 // Build adds the runtime to the imagebuildah call
 func (r *Runtime) Build(ctx context.Context, options buildahDefine.BuildOptions, dockerfiles ...string) (string, reference.Canonical, error) {
 	if options.Runtime == "" {
-		// Make sure that build containers use the same runtime as Podman (see #9365).
-		conf := util.DefaultContainerConfig()
-		options.Runtime = conf.Engine.OCIRuntime
+		options.Runtime = r.GetOCIRuntimePath()
```

Comment:
Could the default runtime ever differ from what's set in the containerConfig structure?  If so, I'm not sure we'd want to do this.

Reply:
They are stored in the same structure.  We read it originally from the containers.conf, but if the user overrides it, we replace the value in the same structure.